pkgs/development/tools/volta/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  rustPlatform,
+  libiconv,
+  stdenv,
+  installShellFiles,
+  darwin,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "volta";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "volta-cli";
+    repo = pname;
```

Comment:
Use the name of the repo instead of a variable.

Reply:
Done